torq.q

```diff
@@ -612,12 +613,11 @@ if[@[value;`.servers.STARTUP;0b]; .servers.startup[]]
 
 // function to execute functions in .proc.initlist
 .proc.init:{
-	$[count .proc.initlist;
-		[{[a].lg.o[`init;"attemping to run initialisation: ",-3!a];
-		@[value;a;
-		{[x;a].lg.e[`init;x," error - failed to run initialisation: ",-3!a]}[;a]]}
-		each .proc.initlist;.proc.initlist:()];
-		.lg.o[`init;"no initialisation functions found"]];
+    if[0=count .proc.initlist;:.lg.o[`init;"no initialisation functions found"]];
+    {[a].lg.o[`init;"attemping to run initialisation: ",-3!a];
+        @[value;a;
+        {[x;a].lg.e[`init;x," error - failed to run initialisation: ",-3!a]}[;a]]}
+        each .proc.initlist;.proc.initexecuted,:.proc.initlist;.proc.initlist:();
```

Comment:
I think this needs cleaned up a bit so it's easier to see where the lambda starts + stops:
```
{[a]
  .log.o...
  @[value;a;{[x;a].lg.e...
}each .proc.initlist;
.proc.initexecuted,:.proc.initlist
.proc.initlist:();
```
Thoughts? @jonnypress + @jonathonmcmurray

Reply:
perhaps this multiline lambda could be made into it's own function `.proc.try` or something like that? so then in this function we would do `.proc.try each .proc.initlist`

I think even with good formatting, multiline lambdas can be confusing and hard to read, best to avoid them wherever we can

Maybe we can make it more generic as well, have it accept (in this case) `` `init``, and remove the word "initialisation " from the log messages; then this could in theory be used in other places if necessary. And here it would be ``.proc.try[`init] each .proc.initlist``